integration-libs/opf/base/opf-api/adapters/opf-api-base.adapter.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Injectable, inject } from '@angular/core';
+import {
+  ConverterService,
+  LoggerService,
+  backOff,
+  isJaloError,
+  normalizeHttpError,
+} from '@spartacus/core';
+import {
+  OPF_ACTIVE_CONFIGURATION_NORMALIZER,
+  OpfBaseAdapter,
+  OpfEndpointsService,
+} from '@spartacus/opf/base/core';
+import {
+  ActiveConfiguration,
+  OPF_CC_PUBLIC_KEY,
+  OpfConfig,
+} from '@spartacus/opf/base/root';
+import { Observable } from 'rxjs';
+import { catchError } from 'rxjs/operators';
+
+@Injectable()
+export class OpfApiBaseAdapter implements OpfBaseAdapter {
+  protected logger = inject(LoggerService);
+
+  constructor(
+    protected http: HttpClient,
+    protected converter: ConverterService,
+    protected opfEndpointsService: OpfEndpointsService,
+    protected config: OpfConfig
+  ) {}
+
+  protected headerWithNoLanguage: { [name: string]: string } = {
+    accept: 'application/json',
+    'Content-Type': 'application/json',
+  };
+  protected header: { [name: string]: string } = {
+    ...this.headerWithNoLanguage,
+    'Accept-Language': 'en-us',
+  };
+
+  protected headerWithContentLanguage: { [name: string]: string } = {
+    ...this.headerWithNoLanguage,
+    'Content-Language': 'en-us',
```

Comment:
[QUESTION]

Is it OK that we hardcode `en-us` value in the adapter? When the end user uses a different language than English, shouldn't we send a different value to the `OpfApi`?

Note: an analogical question applies to all `OpfApi`-adapters

Note 2: I can see, the same logic is copy pasted also in other adapters. **If this logic is likely to be customized by customers**, I'd consider moving it into a shared place, so it's easier to customize just in one place.